wp-content/plugins/core/src/Blocks/Types/Accordion/Accordion.php

```diff
@@ -3,78 +3,95 @@
 
 namespace Tribe\Project\Blocks\Types\Accordion;
 
-use Tribe\Gutenpanels\Blocks\Block_Type_Interface;
-use Tribe\Gutenpanels\Blocks\Sections\Content_Section;
-use Tribe\Gutenpanels\Blocks\Sections\Toolbar_Section;
-use Tribe\Project\Blocks\Block_Type_Config;
-use Tribe\Project\Blocks\Types\Accordion\Support\Accordion_Section;
+use Tribe\Libs\ACF\Field;
+use Tribe\Libs\ACF\Block;
+use Tribe\Libs\ACF\Block_Config;
+use Tribe\Libs\ACF\Repeater;
 
-class Accordion extends Block_Type_Config {
-	public const NAME = 'tribe/accordion';
+class Accordion extends Block_Config {
+	public const NAME = 'accordion';
 
-	public const TITLE       = 'title';
-	public const DESCRIPTION = 'description';
-	public const ACCORDION   = 'accordion';
+	public const TITLE             = 'title';
+	public const DESCRIPTION       = 'description';
+	public const ACCORDION         = 'accordion';
+	public const ACCORDION_HEADER  = 'accordion_header';
+	public const ACCORDION_CONTENT = 'accordion_content';
 
 	public const LAYOUT         = 'layout';
 	public const LAYOUT_INLINE  = 'layout-inline';
```

Comment:
Let's remove the prefix from these values `layout-`. This was a limitation of the GutenPanels options implementation.

Reply:
One thing to keep in mind here is the value must relate to the image. So if we have `left` or `center` then we probably need to specific the path for the images like `/assets/img/admin/blocks/accordion` so they don't conflict with a `left` or `center` specific to another panel.

Reply:
Ah, right. I wasn't thinking that thru. Would be nice to be able to specify an image per option similar to the swatch select's color-per-option. Or, maybe, doing a per-block folder in assets isn't a bad idea anyway. It makes the images directory a bit quicker to grep.

For icons/images that aren't specific, they could remain in the root, but add a block-specific directory for any specific options. Thoughts?